cpp/cmake/Dependencies.cmake

```diff
@@ -61,7 +61,7 @@ set(FAISS_DIR ${CMAKE_CURRENT_BINARY_DIR}/faiss CACHE STRING
   "Path to FAISS source directory")
 ExternalProject_Add(faiss
   GIT_REPOSITORY    https://github.com/facebookresearch/faiss.git
-  GIT_TAG           v1.6.1
+  GIT_TAG           v1.6.2
```

Comment:
Do we need the version update?

Reply:
yeah, wanted to bump the version so it’s up to date so long as there’s no complicated API changes. This new version also brings some of the new lp-based metrics from the scipy package! Unfortunately, there’s a known issue with 1.6.3 that causes build issues. We will probably end up bumping to 1.6.4 when it’s release. Thank you for the quick review!

Reply:
fair enough Corey. Thanks for the description.